mysql_scripts/table_creation/FullDB.sql

```diff
@@ -172,9 +177,7 @@ CREATE TABLE Shift (
 	shiftId INTEGER UNSIGNED PRIMARY KEY NOT NULL AUTO_INCREMENT,
 	startTime DATETIME NOT NULL,
 	endTime DATETIME NOT NULL,
-	handlesResidential BOOLEAN NOT NULL,
-	handlesInternational BOOLEAN NOT NULL,
-	deleted BOOLEAN NOT NULL DEFAULT FALSE,
```

Comment:
Got rid of the handlesResidential / handlesInternational since this was a bad way to set it up since VITA handles 5 different kinds of taxes. A more robust system will have to be made to handle these situations where this site is only handling X taxes these days. But also, she said it's a hard date that they start doing international taxes (Tuesdays/Thursdays in March/April), so we can get away with hardcoding this for now

Renamed deleted to archived to be more standard across the rest of the system